### DISCUSSION TITLE
net6.0 and net7.0

### DISCUSSION THREAD

**Discussion by dtewinkel** [canton7/RestEase#250] [General]
Are there any plans for supporting the latest .NET core versions, such as .NET core 6 and .NET core 7? .NET core 5.x is out of support.

**Comment by canton7** [comment]
They are supported: https://www.nuget.org/packages/restease#supportedframeworks-body-tab

**Comment by canton7** [comment]
For libraries, depending on .NET 5 just means that they require the API surface provided by. NET 5. Later versions of .NET provide at least those same APIs, so a library targeting NET 5 will also run on anything later.

The runtime that your application actually runs on is determined by what the application targets, not its libraries.

**Comment by dtewinkel** [comment]
Thank you for the quick and clear answer.
